spec/person_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../person'
+require 'yaml'
+
+describe Person do
+  before :each do
+    @person = Person.new(25, 'bello', 'teacher')
+  end
+  it 'returns persons name bello' do
+    @person.correct_name.should eql 'bello'
```

Comment:
- [**Optional**] The syntax `should` is deprecated(a warning from the console) hence let's use the new `expect` syntax
- **Hint**: This line can be rewritten as `expect(@person.age).to eql 25`